code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -127,46 +127,55 @@
 	icon_state = "blshell"
 	projectile_type = /obj/item/projectile/bullet/blank
 	starting_materials = list(MAT_IRON = 250)
-	w_type = RECYK_METAL
 
 /obj/item/ammo_casing/shotgun/beanbag
 	name = "beanbag shell"
 	desc = "A weak beanbag shell."
 	icon_state = "bshell"
 	projectile_type = /obj/item/projectile/bullet/weakbullet
 	starting_materials = list(MAT_IRON = 500)
-	w_type = RECYK_METAL
 
 /obj/item/ammo_casing/shotgun/fakebeanbag
 	name = "beanbag shell"
 	desc = "A weak beanbag shell."
 	icon_state = "bshell"
 	projectile_type = /obj/item/projectile/bullet/weakbullet/booze
 	starting_materials = list(MAT_IRON = 12500)
-	w_type = RECYK_METAL
 
 /obj/item/ammo_casing/shotgun/stunshell
 	name = "stun shell"
 	desc = "A stunning shell."
 	icon_state = "stunshell"
 	projectile_type = /obj/item/projectile/bullet/stunshot
 	starting_materials = list(MAT_IRON = 2500)
-	w_type = RECYK_METAL
 
 /obj/item/ammo_casing/shotgun/dart
 	name = "shotgun dart"
 	desc = "A dart for use in shotguns."
 	icon_state = "blshell"
 	projectile_type = /obj/item/projectile/bullet/dart
 	starting_materials = list(MAT_IRON = 12500)
-	w_type = RECYK_METAL
 
 /obj/item/ammo_casing/shotgun/buckshot
 	name = "buckshot shell"
 	desc = "A 12 gauge shell filled with standard double-aught buckshot."
 	icon_state = "bsshell"
 	projectile_type = /obj/item/projectile/bullet/buckshot
 	starting_materials = list(MAT_IRON = 12500)
+	
+/obj/item/ammo_casing/shotgun/dragonsbreath
+	name = "dragon's breath shell"
+	desc = "A 12 gauge shell filled with an incendiary mixture, for lighting up dark areas or setting things on fire."
+	icon_state = "bdbshell"
+	projectile_type = /obj/item/projectile/bullet/fire_plume/dragonsbreath
+	starting_materials = list(MAT_IRON = 12500, MAT_PLASMA = 2000)
+	
+/obj/item/ammo_casing/shotgun/frag
+	name = "explosive shell"
+	desc = "A 12 gauge shell filled with a high-explosive mixture, for heavy anti-personnel usage."
+	icon_state = "fragshell"
+	projectile_type = /obj/item/projectile/bullet/boombullet
+	starting_materials = list(MAT_IRON = 12500, MAT_PLASMA = 8000)
```

Comment:
Seems a bit odd to me that the flammable thing would have less plasma than the explosive thing, but w/e.

Reply:
Highly compressed plasma pellets cause explosions when they expand on impact.